mlflow/ml-package-versions.yml

```diff
@@ -628,7 +628,7 @@ openai:
     install_dev: |
       pip install git+https://github.com/openai/openai-python
   models:
-    minimum: "0.27.2"
+    minimum: "1.0.1"
```

Comment:
autologging minimum version is 1.17.0. I think we chose the latest value when we implemented autologging.